setup.py

```diff
@@ -137,6 +137,7 @@ def _build_dependency(self, aws_lib):
             '-DBUILD_SHARED_LIBS=OFF',
             '-DCMAKE_BUILD_TYPE={}'.format(build_type),
             '-DBUILD_TESTING=OFF',
+            '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
```

Comment:
Does this change actually fix anything? This cmake command is only used to build the `aws-c-*` libraries. The python module is compiled and linked by setup.py magicsauce, not cmake.

Are you trying to solve the issue described in builder.json, where you the chunk about "don't run clang-tidy until we find a way to generate compile_commands.json."?

Reply:
Oh interesting, you're right. I can undo this. I kinda fired that one into the wind while waiting for a build to run.

Reply:
this change still needs to be removed